nflow-engine/src/main/java/io/nflow/engine/workflow/curated/ExecutorMaintenanceWorkflow.java

```diff
@@ -0,0 +1,63 @@
+package io.nflow.engine.workflow.curated;
+
+import static io.nflow.engine.workflow.definition.NextAction.moveToState;
+
+import javax.inject.Inject;
+
+import org.springframework.stereotype.Component;
+
+import io.nflow.engine.service.ExecutorMaintenanceConfiguration;
+import io.nflow.engine.service.MaintenanceService;
+import io.nflow.engine.workflow.definition.NextAction;
+import io.nflow.engine.workflow.definition.StateExecution;
+import io.nflow.engine.workflow.definition.StateVar;
+import io.nflow.engine.workflow.definition.WorkflowSettings;
+
+/**
+ * Clean up workflow executors periodically.
+ */
+@Component
+public class ExecutorMaintenanceWorkflow extends CronWorkflow {
```

Comment:
That would be a different PR yes. Toggle would be nice but Explorer config would be enough IMO.

I was thinking about merging this to existing maintenance workflow, let's see how it would change things.

Reply:
Merged